### DISCUSSION TITLE
WinGet - exclude licensed app from beeing updated

### DISCUSSION THREAD

**Discussion by GhostfaceKilla15** [Romanitho/Winget-Install#103] [Q&A] [answered]
How to disable app updates via winget and Intune ADMX? All Policies set up like described in @Romanitho post. Apps on Blacklist are still getting updated, even if there are entries in the registry or in the configuration settings in Intune.

![Bild (1)](https://github.com/Romanitho/Winget-Install/assets/157619205/9a6cb232-3a7d-430a-969d-4c06f0708e01)

**Comment by KnifMelti** [comment, marked as answer]
Wrong Repo, but:
Have you set the **external Path** to the abbreviation: **GPO**?
This is done this way so that you can shift between some URL (as in our case) and GPO without loosing the list in GPO.

![image](https://github.com/Romanitho/Winget-Install/assets/102996177/5bdbb3c1-5aa8-43cd-a10a-f8e0f982bf7a)

**Comment by GhostfaceKilla15** [comment]
Hey @KnifMelti,
thx for your reply, No,I have not set it like you mentioned. I will check it out and let you know.
